src/test/java/com/ericsson/ei/queryservice/test/QueryServiceTest.java

```diff
@@ -149,19 +149,30 @@ public void processMissedNotificationTest() {
             actual = new ObjectMapper().readTree(missedNotification).path("AggregatedObject");
 
         } catch (Exception e) {
-            log.error(e.getMessage(), e);
+            LOG.error(e.getMessage(), e);
         }
-        log.debug("The result is : " + record.toString());
+        LOG.debug("The result is : " + record.toString());
         assertEquals(record.toString(), actual.toString());
     }
 
+    @Test
+    public void deleteMissedNotificationTest() {
+        Iterable<Document> responseDB = mongoClient.getDatabase(missedNotificationDataBaseName)
+            .getCollection(missedNotificationCollectionName).find();
+        Iterator itr = responseDB.iterator();
+        String response = itr.next().toString();
+        LOG.debug("The inserted doc is : " + response);
+        boolean removed = processMissedNotification.deleteMissedNotification("Subscription_1");
+        assertEquals(true, removed);
```

Comment:
I prefer that you double check with a query. The result from query should be empty.